src/pages/monitoring/overview/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from "react";
+import { MDBCard, MDBCardBody } from "mdb-react-ui-kit";
+import "./style.css";
+import { useStore } from "../../../store";
+
+type Theme = "light" | "dark";
+
+export const Overview = () => {
+  const store: any = useStore();
+  const [theme, setTheme] = React.useState<Theme>(store?.theme);
+
```

Comment:
don't add theme management in separate components but instead manage it for all components from the sidebar only